packages/cache/src/index.ts

```diff
@@ -0,0 +1,161 @@
+import { Redis } from 'ioredis';
+import { LRUCache } from 'lru-cache';
+import { logger } from '@prairielearn/logger';
+import * as Sentry from '@prairielearn/sentry';
+
+const CACHE_KEY_PREFIX = 'prairielearn-cache:';
+
+let cacheEnabled = false;
+let cacheType: 'redis' | 'memory' | 'none';
+let memoryCache: LRUCache<string, string>;
+let client: Redis;
```

Comment:
These need to be properties of the class. Otherwise, if one creates multiple instances of `Cache`, the latter `init()` calls will overwrite these properties for *all* instances.

Reply:
Ah, good catch. This is updated in [cc33414](https://github.com/PrairieLearn/PrairieLearn/pull/9360/commits/cc33414906007727a55854776b4e89552b90b0dd).